src/Loader.php

```diff
@@ -93,7 +93,7 @@ private function load_services( $services ) {
 				);
 			}
 
-			// Load P4 Metaboxes only when adding/editing a new Page/Post/Campaign.
+			// Load Campaigns only when adding/editing a new Tag.
```

Comment:
Just to clarify the reference of Campaigns::class is not related to the P4CG or "Campaign Pages" here, At very beginning of P4 (when we don't have P4CG as a post type) we were refer the tags to campaign ( [eg](https://github.com/greenpeace/planet4-master-theme/blob/master/src/Search.php#L747) )
and [Category to Issue](https://github.com/greenpeace/planet4-master-theme/blob/master/src/Search.php#L734)

Reply:
Oups, thanks for that, I'll fix the comment 👍